jobsdb/jobsdb.go

```diff
@@ -2058,22 +2058,15 @@ func (jd *HandleT) backupTable(backupDSRange dataSetRangeT, isJobStatusTable boo
 			panic(err)
 		}
 
-		var rows []interface{}
-		err = json.Unmarshal(rawJSONRows, &rows)
-		if err != nil {
-			panic(err)
-		}
-		contentSlice := make([][]byte, len(rows))
-		for idx, row := range rows {
-			rowBytes, err := json.Marshal(row)
-			if err != nil {
-				panic(err)
-			}
-			contentSlice[idx] = rowBytes
+		if len(rawJSONRows) < 2 {
```

Comment:
This is just a safe check so that subslice condition below doesn't panic.
Ideally this should never happen. We have a good number of checks to make sure that the number of rows returned are non zero.

row.Scan for the query `select json_agg (select *...) ....` throws some other error (not ErrNoRows)

Reply:
removed this check.